doc/01-About.md

```diff
@@ -0,0 +1,55 @@
+# Icinga Notifications
+
+!!! warning
+
+    This is an early preview version for you to try, but do not use this in production.
+    There may still be severe bugs and incompatible changes may happen without any notice.
+    At the moment, we don't yet provide any support for this.
+
+Icinga Notifications is a set of components that processes received events from various sources, manages incidents and
+forwards notifications to predefined contacts, consisting of:
+
+* The Icinga Notifications daemon, which receives events and sends notifications.
+* The [Icinga Notifications Web](https://icinga.com/docs/icinga-notifications-web/latest/doc/01-About/) module,
+  which provides graphical configuration.
+* Icinga 2 or other sources that provide monitoring events that result in notifications.
+
+## Big Picture
+
+<!-- TODO: add a real picture -->
+
+Because Icinga Notifications consists of several components,
+this section tries to help understand how these components relate.
+
+First, the Icinga Notifications configuration resides in a SQL database.
+It can be conveniently tweaked via Icinga Notifications Web directly from a web browser.
+The Icinga Notifications daemon uses this database to read the current configuration.
+
+As in any Icinga setup, all host and service checks are defined in Icinga 2.
+By querying the Icinga 2 API, the Icinga Notifications daemon retrieves state changes, acknowledgements, and other events.
+These events are stored in the database and are available for further inspection in the Icinga Notifications Web.
+
+Depending on its configuration, the daemon will take further action on these events.
```

Comment:
further ~action~ actions on ..

Reply:
I am quite certain that it should be _action_ in this case, as "take action" is a collocation referring to some generic action to be "taken care of" and not a specific action, or a multitude of actions.

https://dictionary.cambridge.org/dictionary/english/take-action

Reply:
Given that the demon doesn't just take a single additional action, but quite a few, the **action** somehow sounds wrong to my ears.